packages/algob/test/builtin-tasks/sign-multisig.ts

```diff
@@ -1,17 +1,17 @@
 import { compareArray } from "@algo-builder/runtime/src/lib/compare";
-import { decodeAddress, decodeSignedTransaction } from "algosdk";
+import { decodeAddress, decodeObj, decodeSignedTransaction, decodeUnsignedTransaction } from "algosdk";
 import { assert } from "chai";
 import * as fs from "fs";
 import path from "path";
 import sinon from 'sinon';
 
 import { TASK_SIGN_MULTISIG } from "../../src/builtin-tasks/task-names";
 import { ASSETS_DIR } from "../../src/internal/core/project-structure";
-import { loadSignedTxnFromFile } from "../../src/lib/files";
+import { loadEncodedTxFromFile } from "../../src/lib/files";
 import { HttpNetworkConfig } from "../../src/types";
 import { getEnv } from "../helpers/environment";
 import { useFixtureProject } from "../helpers/project";
-import { account1, bobAcc } from "../mocks/account";
+import { account1, aliceAcc, bobAcc } from "../mocks/account";
```

Comment:
shall we use `aliceAcc.address` in line 24? Otherwise we have a linter error that the variable is not used.

Reply:
yes, thanks. We can use `bobAcc.addr` as well